tests/data/vocabulary_test.py

```diff
@@ -300,3 +300,12 @@ def test_from_params(self):
         # present apart from 'vocabulary_directory' and we aren't calling from_dataset.
         with pytest.raises(ConfigurationError):
             _ = Vocabulary.from_params(Params({"directory_path": vocab_dir, "min_count": {'tokens': 2}}))
+
+    def test_vocab_can_print(self):
+        vocab = Vocabulary(non_padded_namespaces=["a", "c"])
+        vocab.add_token_to_namespace("a0", namespace="a")
+        vocab.add_token_to_namespace("a1", namespace="a")
+        vocab.add_token_to_namespace("a2", namespace="a")
+        vocab.add_token_to_namespace("b2", namespace="b")
+        vocab.add_token_to_namespace("b3", namespace="b")
+        print(vocab)
```

Comment:
You're really testing the `__str__` method here, and it'd be more informative to make sure that the string representation has a few key things that you care about.  i.e., instead of just calling `print(vocab)`, do some `contains` asserts on `str(vocab)`.

Reply:
I'm not really testing it's content, just that it doesn't crash. It's the same as all the other __str__ tests and seems a bit fine grained to test whether a string get's mutated from its string representation in between being called and printing it.

Reply:
The reason I brought it up was because my initial thought was to say "Remove print statement" and move on.  But then I realized the print was actually part of the test.  But you don't _need_ the `print` for the test, just `str(vocab)`.